src/openfluid/config.hpp.in

```diff
@@ -130,6 +130,9 @@ const std::string BUILDEREXTS_WARESBIN_STD_PATH =
     "@OFBUILD_LIB_INSTALL_PATH@/@OPENFLUID_MAIN_NAME@/@OPENFLUID_BUILDEREXTS_PATH@";
 
 
+// Fragments dirs
+const std::string FRAGMENTS_PATH = "@OPENFLUID_FRAGMENTS_PATH@";
```

Comment:
Variable to rename `WARESDEV_SRC_FRAGMENTSDIR` and move it near line 164

Reply:
This variable name has a wider use than the name of the subdirectory in the ware source, it is also the location name used in the `waresdev/` directory  (and for workspace) for shared fragments, which also explains why it is line 49 in CMake.in.cmake.

Reply:
OK, in this case, the correct name would be `WARESDEV_FRAGMENTS_DIR` as fragments are only related to wares development, and it is just a directory name (not a path). And it should be moved to the "waresdev" part of this file for better grouping and consistency of variables.
(and I am totally aware that a full review of variables naming in this file should be done, as there are
 some consistency errors)